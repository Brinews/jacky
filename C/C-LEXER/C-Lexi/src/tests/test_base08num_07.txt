

   1234X08

   0x08
