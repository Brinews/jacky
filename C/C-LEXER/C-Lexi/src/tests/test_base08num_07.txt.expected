{1234 , NUM , 3}
{X08 , ID , 3}
{0x08 , BASE08NUM , 5}
{ , END_OF_FILE , 7}
